.github/workflows/pytest.yaml

```
@@ -0,0 +1,38 @@
+    - name: Install bedtools
+      run: |
+        wget https://github.com/arq5x/bedtools2/releases/download/v2.31.0/bedtools.static -O bedtools
+        chmod a+x bedtools
+        sudo mv bedtools /usr/bin/bedtools
+
+    - name: Test with pytest
+      run: |
+        pytest -v --cov --count 10 --random-order tests/
+
+    - name: Get test coverage % from pytest output for coverage badge
+      run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:4:26: Double quote to prevent globbing and word splitting

(shellcheck)